### DISCUSSION TITLE
BLE iOS vs Android device address

### DISCUSSION THREAD

**Discussion by wesoos** [shinyorg/shiny#966] [Q&A] [answered]
Hello, it looks like the peripheral address is different between iOS and Android.  Is that correct?

Is there a property I can utilize to get the same unique ID between the 2 platforms?

Thanks

**Comment by aritchie** [comment, marked as answer]
No unfortunately.  Due to the way iOS handles things.  You are best to use the device name if you are trying to identify devices.

**Reply by wesoos** [reply]
Thank you

**Reply by wesoos** [reply]
Hi Allan,  is it the case that different versions of IPhones will give me different UUIIDs?  I don't think it's anything in this library causing that, but just want to confirm.

We can't use the name of the device since our use case can have more than one device in range...

**Reply by aritchie** [reply]
Yes - this is known.  Please review the Apple Documentation for information like this.  There are many ways to accomplish device identification if you control the peripheral firmware.
